designs/index.md

```diff
@@ -7,25 +7,248 @@ title: Design Documents
+to get feedback from Bazel end-users).
+
+
+### Iterate with reviewers
+
+Anyone interested can chime in and comment on the proposal. The author should
+try to answer questions, clarify the proposal, and address the concerns.
+
+Discussion should happen on the announcement thread. If the document is a Google
+Doc, comments may be used instead (but note that anonymous comments are
+allowed).
+
+
+### Update the status
+
+When the author believes the iteration round is complete, they send a new PR to
+update the status. The PR must be sent to the lead reviewer and cc the other
+reviewers.
+
+The lead reviewer approves the PR to officially accept or reject the proposal.
+It is the lead reviewer's responsibility to ensure that other reviewers agree
+with the decision.
+
+There must be at least 1 week between the first announcement and the approval of
+a proposal. This ensures that users had enough time to read the document and
+share their concerns.
+
+
+## Should I use markdown or Google Docs?
+
+Both are accepted. The author can decide what works best for them.
+
+Google Docs can be more effective for brainstorming, collaborative editing, and
+quick iteration. Suggested edits are also very valuable.
+
+Markdown files have some other benefits, including:
+
+*   Clean URLs for linking.
+*   Explicit record of revisions.
+*   No forgetting to set up access rights before publicizing a link.
+*   More easily searchable with search engines.
+*   Future-proofness: plain text is not at the mercy of any specific tool,
+    doesn't require an Internet connection.
+*   It is possible to update them even if the author is not around anymore.
+*   They can be processed automatically (e.g. update/detect dead links, fetch
+    list of authors, etc.).
+
+It is also possible to first iterate on a Google Doc, and then convert it to
+markdown for posterity.
+
+
+## Using Google Docs
+
+To make a [Google Doc](https://docs.google.com) world-readable, create it with a
+personal account. To share it, click on "Share", "Advanced", then "Change…", and
+choose "On - Anyone with the link". You may allow comments on the document. If
+you do so, anyone will be able to comment anonymously, even without a Google
+account.
+
+
+## Using Markdown
+
+Documents are stored on GitHub and use the GitHub flavor of Markdown
+([overview](https://guides.github.com/features/mastering-markdown/),
+[specification](https://github.github.com/gfm/)).
+
+Send a PR to update an existing document. Significant changes should be reviewed
+by the document reviewers. Trivial changes (e.g. typos, formatting) can be
+approved by anyone.
+
+
+## How to choose a lead reviewer?
+
+The lead reviewer is a  domain expert. Lead reviewers must be:
+
+*   Knowledgeable of the relevant subsystems
+*   Objective (i.e., capable of providing constructive feedback)
+*   Have sufficient time available to lead the review process
```

Comment:
Lead reviews must be: available for the entire review period to lead the process